ImagesTable: add option to download a raw compose request

ondrejbudai opened the pull request:
In order to help people transition from using GUI to use the API directly,
it's helpful to give them an easy way to inspect the whole raw compose
request.

This commit adds a new button next to each compose that simply downloads
the original compose request in a json format. This request can then be
directly piped into the IB API to build a new image.

## Notes

Manipulation DOM directly when using React? Crazy, right? It works though but I'm open to suggestions.

## Demo
https://www.youtube.com/watch?v=pr-6evaue_4
(I failed to embed gif, sry.)

## Static picture
![image](https://user-images.githubusercontent.com/119861/182806879-58b6fac5-a102-4f04-ad5d-70af6166cc79.png)




Original idea by Troy Dawson

lucasgarfield commented on src/Components/ImagesTable/ImagesTable.js:
```diff
@@ -128,6 +128,25 @@ const ImagesTable = () => {
         }),
     },
+    {
+      title: 'Download compose request (.json)',
```
It's actually possible to use anchor tags in the action title, so we could do something like this if we want to avoid manipulating the DOM:
![image](https://user-images.githubusercontent.com/95542540/182816785-430f1cfb-d2b9-42fa-b4ae-8bf8afb6544f.png)

ondrejbudai replied:
Isn't the formatting different though? 
![image](https://user-images.githubusercontent.com/119861/182822558-e8167a7c-d8de-45ed-b921-86f4edace718.png)

lucasgarfield replied:
We could use css to adjust the formatting, however, it might be okay as is.

Generally we try to keep things as close to the PatternFly examples as possible. Consider this example:
https://www.patternfly.org/v4/components/table#composable-actions

The link is blue with underlining on over since it is an anchor tag. And it makes sense to use an anchor tag for this purpose, downloading a resource, in my opinion.

In any case, the design team will have the final say and we can just ask them what they want us to do!

ondrejbudai commented:
Rewritten in the way @lucasgarfield suggested (and credited him) and added a simple test.

This is how the button looks now:
![image](https://user-images.githubusercontent.com/119861/184606845-5b9e8c87-6bcc-4343-808c-ee295d83ecb1.png)

Gundersanne reviewed (commented):
Needs a `npm run lint:js:fix` but other than that lgtm :)

ondrejbudai commented:
Fixed, thanks!

Gundersanne reviewed (approved):
lgtm! thank you
